base/dft.jl

```diff
@@ -395,8 +395,17 @@ if Base.USE_GPL_LIBS
 
     A multidimensional FFT simply performs this operation along each transformed dimension of `A`.
 
-    Higher performance is usually possible with multi-threading. Use `FFTW.set_num_threads(np)`
-    to use `np` threads, if you have `np` processors.
+    Notes
```

Comment:
there's a specific markdown syntax we can use for notes now, see https://github.com/JuliaLang/julia/pull/17272

this would also need a `make docs` and commit the rst